scripts/quests/ahtUrhgan/Ode_to_the_Serpents.lua

```diff
@@ -0,0 +1,76 @@
+-----------------------------------
+-- Ode to the Serpents
+-----------------------------------
+-- Log ID: 6, Quest ID: 44
+-- Fari-Wari: !pos 80 -6 -137 50
+-----------------------------------
+require("scripts/globals/items")
+require("scripts/globals/quests")
+require("scripts/globals/npc_util")
+require('scripts/globals/interaction/quest')
+require("scripts/globals/zone")
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.ODE_TO_THE_SERPENTS)
+
+quest.reward =
+{
+    item = xi.items.IMPERIAL_GOLD_PIECE,
+}
+
+quest.sections =
+{
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getQuestStatus(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.SAGA_OF_THE_SKYSERPENT) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Fari-Wari'] = quest:progressEvent(882),
+
+            onEventFinish =
+            {
+                [882] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    npcUtil.giveKeyItem(player, xi.ki.BIYAADAS_LETTER)
+                end,
+            },
+        },
+    },
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.AHT_URHGAN_WHITEGATE] =
+        {
+            ['Fari-Wari'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        player:getQuestStatus(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.WHEN_THE_BOW_BREAKS) == QUEST_COMPLETED
+                        and player:getQuestStatus(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.FIST_OF_THE_PEOPLE) == QUEST_COMPLETED
```

Comment:
hasCompleted quest, also styling guide.  `and` should be at the end of the first condition